client/state/stats/lists/selectors.js

```diff
@@ -119,3 +120,28 @@ export function getSiteStatsPostsCountByDay( state, siteId, query, date ) {
 	return data[ date ] || null;
 }
 
+/**
+ * Returns a parsed object of statsPublicize data for a given query, or default "empty" object
+ * if no statsStreak data has been received for that site.
+ *
+ * @param  {Object}  state    Global state tree
+ * @param  {Number}  siteId   Site ID
+ * @param  {Object}  query    Stats query object
+ * @return {Array}            Parsed Data for the query
+ */
+export const getSiteStatsParsedData = createSelector(
+	( state, siteId, statType, query ) => {
+		const data = getSiteStatsForQuery( state, siteId, statType, query );
+
+		if ( 'function' === typeof Parsers[ statType ] ) {
+			return Parsers[ statType ].call( this, data );
+		}
```

Comment:
What are the other values that `statType` can take here, besides `'stats'` as in this PR?  Do you anticipate needing to perform different operations for different `statType`s?